kolibri/plugins/user/assets/src/views/SignUpPage.vue

```diff
@@ -40,12 +40,12 @@
             :disabled="busy"
           />
 
-          <template v-if="currentFacility">
+          <template v-if="selectedFacility.name">
```

Comment:
Need to add a "Sign in with an existing account" escape link at the bottom of this form. (See last step of Scenario 1 on Figma)